### DISCUSSION TITLE
How Run RustDesk From Delphi Application

### DISCUSSION THREAD

**Discussion by alibi33** [rustdesk/rustdesk#4638] [Q&A]
Delphi 11.3
Rustdesk 1.1.9.0

How could I run RustDesk from my Delphi application?

This way does not work to run RustDesk.

procedure RustDesk;
var
   File: string;
begin
   File := ExtractFilePath(Application.ExeName) + 'RustDesk.exe';
   ShellExecute(0, nil, PChar(File), nil, nil, 0);
end;

I tested with WinExec and CreateProcess, without success.

Any suggestion?

**Comment by SergeyMy** [comment]
_function ShellExecute(hWnd: [HWnd]; Operation, FileName, Parameters, **Directory**: PChar; ShowCmd: Integer): HINST; stdcall;_
specify the path where the rustdesk libraries are

PS you write a program for scammers? 
ShowCmd = SW_HIDE  0
